src/output/dnssim.c

```diff
@@ -136,6 +138,12 @@ void output_dnssim_free(output_dnssim_t* self)
     free(self);
 }
 
+void output_dnssim_log_name(output_dnssim_t* self, const char* name)
+{
+    snprintf(self->_log.name, LOG_NAME_MAXLEN, "%s", name);
+    self->_log.is_obj = false;
```

Comment:
Why are you setting this to false?

Reply:
Because I don't want the pointer name printed in the log. It's just noise when instances have a unique name set.

Reply:
Document that in `DnsSim:log(name)` then.

Reply:
Added a mentioned that log names should be unique. Feel free to change that if you think something else should be mentioned.